Package.swift

```diff
@@ -19,7 +19,7 @@ let package = Package(
         .package(url: "https://github.com/tuist/XcodeProj.git", from: "7.18.0"),
         .package(url: "https://github.com/jakeheis/SwiftCLI.git", from: "6.0.0"),
         .package(url: "https://github.com/mxcl/Version", from: "2.0.0"),
-        .package(url: "https://github.com/SwiftDocOrg/GraphViz.git", from: "0.4.0"),
+        .package(url: "https://github.com/SwiftDocOrg/GraphViz.git", from: "0.1.0"),
```

Comment:
To fix the original issue in https://github.com/yonaskolb/XcodeGen/pull/1065, should this be pinned to 0.2?

Reply:
pinned version in Package.resolved is 0.2.0 , so both 0.1.0 and 0.2.0 are available.

Reply:
Yes, but I think `Package.resolved` isn't used if the package is used as a dependency in another package, which is when the issue occurs

Reply:
oh, I got it! thanks for your suggestion. I'll fix now